source/transfer_list.rst

```diff
@@ -657,5 +659,102 @@ such that the last ACPI table in this entry ends at offset
      - hdr_size
      - One or more ACPI tables.
 
+
+.. _tpm_evlog_entry:
+
+TPM event log table entry layout (XFERLIST_EVLOG)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+This entry type holds TPM-related information for a platform. The TPM event log
+info is a region containing a TPM event log as defined by TCG EFI Protocol
+Specification (https://trustedcomputinggroup.org/resource/tcg-efi-protocol-specification/).
+
+.. _tab_tpm_evlog:
+.. list-table:: TPM event log type layout
+   :widths: 2 2 2 8
+
+   * - Field
+     - Size (bytes)
+     - Offset (bytes)
+     - Description
+
+   * - tag_id
+     - 0x3
+     - 0x0
+     - The tag_id field must be set to **5**.
+
+   * - hdr_size
+     - 0x1
+     - 0x3
+     - |hdr_size_desc|
+
+   * - data_size
+     - 0x4
+     - 0x4
+     - The size of the event log in bytes + sizeof(flags) i.e. 0x4.
+
+   * - flags
+     - 0x4
+     - 0x8
```

Comment:
This fixed offset is not robust to changes in the TE header size. It should be hdr_size like other TEs.